automates/text_reading/src/test/scala/org/clulab/aske/automates/text/TestFunctions.scala

```diff
@@ -314,9 +365,92 @@ class TestFunctions extends ExtractionTest {
     testFunctionEvent(mentions, desired)
   }
 
+  // Tests from 1985-description and performance of CERES-Wheat_A_user-oriented wheat yield model
+
+  val t1i = "The primary variable influencing development rate is temperature."
+  failingTest should s"find functions from t1i: ${t1i}" taggedAs(Somebody) in {
+    val desired = Seq(
+      "development rate" -> Seq("temperature") //note: overlap between trigger and output??
+    )
+    val mentions = extractMentions(t1i)
+    testFunctionEvent(mentions, desired)
+  }
+
+  // Tests from paper: PT-1972-On the Assessment of Surface Heat Flux and Evaporation using Large-Scale Parameters-petpt
+
+  val t1j = "The rate of conduction of heat near the interface between two media is governed by the quantity pc√K pertaining to each medium."
+  failingTest should s"find functions from t1j: ${t1j}" taggedAs(Somebody) in {
+    val desired = Seq(
+      "The rate of conduction of heat near the interface between two media" -> Seq("quantity pc√K pertaining to each medium") //todo: identifier within Phrase should be filtered out
```

Comment:
pc√K? If yes, I think it's fine to keep it.

Reply:
Yes that's correct. Thanks for checking!